.github/workflows/java-ec2-default-test.yml

```diff
@@ -239,7 +249,6 @@ jobs:
           aws-region: ${{ env.E2E_TEST_AWS_REGION }}
 
       - name: Save test results
-        if: always()
```

Comment:
Why? It wasn't removed in the ASG version of this test.

Reply:
Remove by accident, readded